src/generic-container.test.ts

```diff
@@ -149,4 +149,15 @@ describe("GenericContainer", () => {
       await startedContainer.stop();
     });
   });
+
+  it("should build and start named container from a Dockerfile", async () => {
```

Comment:
There's a `describe` block for `fromDockerfile` functionality

Reply:
Must be a merge conflict, just moved it to the right place :)